.travis.yml

```diff
@@ -10,8 +10,8 @@ addons:
       - shellcheck
 
 script:
-  - shellcheck *.sh src/*.sh
-  - tests/run_all.sh
+  - shellcheck **/*.sh
+  #- tests/run_all.sh
```

Comment:
Gonna branch off your branch and see if I can't get unit tests working before merge if possible